howto/install/customise.md

```diff
@@ -1,192 +1,60 @@
-The [Deploy Anbox Cloud with Juju](https://discourse.ubuntu.com/t/installation-quickstart/17744) documentation explains how to perform a quick and easy general install of Anbox Cloud. However, in some cases it may be useful to customise the installation:
+There are two main ways to customise the Anbox Cloud deployment:
 
-* Deploying on bare metal
-* Adding additional components
-* Configuring storage or networking
-* Copying an existing configuration
-* Testing a pre-release version
+1. [Use overlays](#overlay-files) in conjunction with the published Anbox Cloud bundle.
+2. [Edit the bundle file](#edit-bundle) itself.
 
-...and many more. This part of the documentation will help you with this.
+Using an overlay means that you can easily apply your customisation to different versions of the bundle, with the possible downside that changes in the structure of new versions of Anbox Cloud might render your overlay obsolete or non-functional (depending on what exactly your overlay does).
 
-## What You Will Need
+Saving a copy of the bundle file and editing that file means that your customisation will always work, but of course, this method requires that you create a new file for each version of Anbox Cloud.
 
-The rest of this page assumes you already have Juju installed and have added credentials for a cloud and bootstrapped a controller. If you still need to do this, please take a look at the [Deploy Anbox Cloud with Juju](https://discourse.ubuntu.com/t/installation-quickstart/17744), or, for custom clouds (OpenStack, MAAS), please consult the [Juju documentation](https://jaas.ai/docs). In case that you want to deploy on bare metal, please have a look below.
+<a name="overlay-files"></a>
+## Use overlay files
 
-Before you start, please make sure you read about the [requirements](https://discourse.ubuntu.com/t/installation-requirements/17734) Anbox Cloud has.
+A *bundle overlay* is a fragment of valid YAML that is dynamically merged on top of a bundle file before deployment, similar to a patch file. The fragment can contain additional or alternative YAML that is intelligible to Juju. For example, to specify custom instance types for the machines used in your cloud of choice, you could use the following fragment:
 
```

Comment:
Can we add a note here saying that the machine constraints below are just an example but not necessarily meant to be use for production? Sizing for production needs different considerations.